app/src/main/java/org/simple/clinic/di/network/NetworkModule.kt

```diff
@@ -84,7 +86,9 @@ class NetworkModule {
         .add(Outcome.MoshiTypeAdapter())
         .add(ContactType.MoshiTypeAdapter())
         .add(QuestionnaireType.MoshiTypeAdapter())
-        .add(QuestionnaireLayoutJsonAdapter().getFactory())
+        .add(OrientationType.MoshiTypeAdapter())
+        .add(InputFieldType.MoshiTypeAdapter())
```

Comment:
Why do we need to add orientation and input field here separately? (And not the other types like sub-header, separator etc)